src/fusor/translator.py

```diff
@@ -269,50 +269,57 @@ async def from_star_fusion(
         )
 
     async def from_fusion_catcher(
-        self, fc_row: pl.DataFrame, rb: Assembly
+        self,
+        five_prime_partner: str,
+        three_prime_partner: str,
+        five_prime_fusion_point: str,
+        three_prime_fusion_point: str,
+        predicted_effect: str,
+        rb: Assembly,
     ) -> AssayedFusion:
         """Parse FusionCatcher output to create AssayedFusion object
 
-        :param fc_row: A row of FusionCatcher output
+        :param five_prime_partner: Gene symbol for the 5' fusion partner
+        :param three_prime_partner: Gene symbol for the 3' fusion partner
+        :param five_prime_fusion_point: Chromosomal position for the 5' end of the
+        fusion junction. This coordinate is 1-based
+        :param three_prime fusion_point:  Chromosomal position for the 3' end of the
+        fusion junction. This coordinate is 1-based
+        :param predicted_effect: The predicted effect of the fusion event, created
+        using annotation from the Ensembl database
         :param rb: The reference build used to call the fusion
         :return: An AssayedFusion object, if construction is successful
         """
-        gene1 = fc_row.get_column("Gene_1_symbol(5end_fusion_partner)").item()
-        gene2 = fc_row.get_column("Gene_2_symbol(3end_fusion_partner)").item()
-        gene_5prime = self._get_gene_element(gene1, "fusion_catcher")[0].gene.label
-        gene_3prime = self._get_gene_element(gene2, "fusion_catcher")[0].gene.label
+        gene1 = five_prime_partner
+        gene2 = three_prime_partner
+        gene_5prime_element = self._get_gene_element(gene1, Caller.FUSION_CATCHER)[0]
+        gene_3prime_element = self._get_gene_element(gene2, Caller.FUSION_CATCHER)[0]
```

Comment:
I think this is cleaner since gene1 and gene2 aren't used any place else in the method
```suggestion
        gene_5prime_element = self._get_gene_element(five_prime_partner, Caller.FUSION_CATCHER)[0]
        gene_3prime_element = self._get_gene_element(three_prime_partner, Caller.FUSION_CATCHER)[0]
```